### ISSUE TITLE
Icon Library page updates

### ISSUE THREAD

**Issue by alisonjoseph** [carbon-design-system/design-language-website#125]
- [x] Grid is not right. The right column of icon containers goes into margin.
- [x]  Remove resource cards over search
- [ ]  Search should be sticky (Shixie has design)
- [x]  Start with 32x32 (reverse order). I think it’s more helpful to be able to see them clearly in the library. 

*copy over latest Icon library component from Carbon as starting point.*

**Comment by emyarod** [comment]
all that's left for this is the sticky search design handoff right?

**Comment by alisonjoseph** [comment]
@emyarod for now at least, I think the plan will be to bring over some of the advanced search stuff the Icon team is building for the Carbon website. There isn't anything else we need to do right now related to Icon library though

**Comment by alisonjoseph** [comment]
Going to close this and defer to updates from Icon project and their issues. We can reopen a new issue once the component is ready to be moved over
